### ISSUE TITLE
vscode-insiders Can't install pre-release version of 'redhat.vscode-yaml' extension

### ISSUE THREAD

**Issue by tjx666** [redhat-developer/vscode-yaml#819]
## Describe the bug

```
Can't install pre-release version of 'redhat.vscode-yaml' extension because it is not compatible with the current version of Visual Studio Code - Insiders (version 1.72.0-insider).
```

![image](https://user-images.githubusercontent.com/41773861/188349623-2d499df3-7604-4de3-8ab9-3bbfbd649b50.png)

## Expected Behavior

Can install pre-release

## Current Behavior

Can't and notify error message

## Steps to Reproduce

1. open extension sidebar view
2. click this extension item
3. click `Switch to Pre-Release Version` button

## Environment
- [ ] Windows
- [x] Mac
- [ ] Linux
- [ ] other (please specify)

```
Version: 1.72.0-insider (Universal)
Commit: c4722f1bb292284e0437eb7547808a5788fe6430
Date: 2022-09-02T05:20:22.688Z
Electron: 19.0.12
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Darwin x64 21.6.0
Sandboxed: Yes
```

**Comment by asampal** [comment]
I'm having the same problem. Maybe this is why the Microsoft Kubernetes extension (which depends on this) is also not working.

**Comment by msivasubramaniaan** [comment]
The fix was available with the latest pre-release v1.11.10112022
